.gitlab-ci.yml

```
@@ -12,7 +12,7 @@ stages:
 
 # define which docker image to use for builds
 default:
-  image: bigbluebutton/bbb-build:v3.0.x-release--2025-09-16-174734
+  image: bigbluebutton/bbb-build:grails7ver2--2025-10-02-015250
```

```suggestion
  image: bigbluebutton/bbb-build:grails7ver2--2025-10-29-110043
```

I just re-did the dockerfile with grails 7.0.0